services/app/lib/codebattle_web/router.ex

```diff
@@ -32,6 +32,7 @@ defmodule CodebattleWeb.Router do
     scope "/v1", V1, as: :v1 do
       get("/:user_id/activity", ActivityController, :show)
       get("/game_activity", GameActivityController, :show)
+      get("/user/:id/stats", UserController, :get_stats)
```

Comment:
We don't like to use custom actions like `:get_stats`.
You've duplicated information, `get` it is http verb, `stats`  you use in url. 
So the better way is to use  `:show`  action. 
Like [resources in phoenix routing](https://hexdocs.pm/phoenix/routing.html#resources)